corehq/apps/sms/templates/sms/whatsapp_templates.html

```diff
@@ -0,0 +1,49 @@
+{% extends "hqwebapp/base_section.html" %}
+{% load i18n %}
+{% load hq_shared_tags %}
+
+{% block title %}
+  {% trans "WhatsApp Templates" %}
+{% endblock %}
+
+
+{% block page_content %}
+  <h2>{% trans "WhatsApp Templates" %}</h2>
+  <hr/>
+  <p>
+    {% blocktrans %}
+      To send a template message, copy and paste the <strong>Template String</strong> as the message text, replacing each <code>{var}</code> with the variables you want to add in to the template. Don't forget the <code>~</code> at the end.
```

Comment:
What happens if the user mis-formats this, for example leaves the tilde off?

Reply:
If you leave off the tilde nothing happens except if you are using a multiple choice question, where Form Player adds in the choices at the end. The choices (e.g. `1: Yes, 2:no`) would then be added in as variables at the end of the list.

Reply:
I think you've explained this at least 3 times to me. Thanks for the patience.